File: src/main/java/com/server/computerscience/question/common/service/ExternalQuestionService.java
package com.server.computerscience.question.common.service;

import org.springframework.stereotype.Service;

import com.server.computerscience.chatbot.dto.response.ChatGptFileUploadResponseDto;
import com.server.computerscience.question.common.dto.request.RequestQuestionCommandDto;

@Service
public interface ExternalQuestionService {
	public ChatGptFileUploadResponseDto sendQuestionToExternal(RequestQuestionCommandDto requestQuestionCommandDto);
}

File: application/src/main/java/com/api/computerscience/question/major/admin/service/ExternalQuestionService.kt
package com.api.computerscience.question.major.admin.service

import com.core.computerscience.chatbot.dto.response.ChatGptFileUploadResponseDto
import com.persistence.computerscience.question.common.domain.dto.request.RequestQuestionCommandDto

@org.springframework.stereotype.Service
interface ExternalQuestionService {
	fun sendQuestionToExternal(requestQuestionCommandDto: RequestQuestionCommandDto): ChatGptFileUploadResponseDto?
}
